src/main/java/filter/SecurityFilter.java

```diff
@@ -0,0 +1,45 @@
+package filter;
+
+import javax.servlet.*;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Actually this class is not used
+ */
+public class SecurityFilter implements Filter {
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
+        HttpServletRequest req = (HttpServletRequest) request;
+        HttpServletResponse res = (HttpServletResponse) response;
+
+        boolean logged = false;
+
+        Cookie[] cookies = req.getCookies();
+        if (cookies != null) {
+            for (Cookie cookie : cookies) {
+                if ("logged".equals(cookie.getName()) && "true".equals(cookie.getValue())) {
```

Comment:
use Boolean.valueOf() instead of equals
and make some new variable instead of hard-coded "logged" and etc.

Reply:
Yeah, but I was going to do the filters before the final assembly of the project